src/index.tsx

```diff
@@ -1,17 +1,28 @@
+import {
+  Endpoint,
+  useEndpoints,
+} from '@jellyfish-dev/react-native-membrane-webrtc';
+
 export * from './JellyfishClientContext';
 export {
-  useRoomParticipants,
   useAudioSettings,
-  useAudioTrackMetadata,
-  useCameraState,
-  useMicrophoneState,
+  updateAudioTrackMetadata,
+  useCamera,
+  useMicrophone,
   useScreencast,
-  useVideoTrackMetadata,
-  useSimulcast,
+  updateVideoTrackMetadata,
   useBandwidthEstimation,
-  useBandwidthLimit,
+  updateEndpointMetadata,
```

Comment:
this should be peer to, and any user facing functions too

Reply:
Yeah, sorry for that I forgot that there could be other methods with this name. Should be all renamed now.